yoti/views/admin-options.php

```diff
@@ -4,6 +4,12 @@
  * @var string $updateMessage
  * @var array $errors
  */
+// Check link users by email address
```

Comment:
My idea was to remove this feature altogether. Would anyone want to have it unchecked? It seems the code sets the checkbox to checked, but I feel like `yoti_user_email` should also be set to TRUE by default. Is this happening?

Reply:
Yes I think we should provide an option to unset `yoti_user_email` in case some people will want to unset it.
Also once saved, the value of `yoti_user_email` will be set to 1 which is the same as TRUE. This works the same way as setting default value for `yoti_success_url` in Drupal 7 for instance.
I am not sure we should force admin users to use this functionality.